docs/tuf-spec.txt

```diff
@@ -268,7 +268,12 @@ Version 1.0 (Draft)
    configured repository.
 
    The root role's private keys must be kept very secure and thus should be
-   kept offline.
+   kept offline.  If less than a threshold of Root keys are compromised, the
+   repository should revoke trust on the compromised keys.  This can be
+   accomplished with a normal rotation of root keys, covered in section 6.1
+   (key management and migration). If a threshold of root keys is compromised,
+   the Root keys should be updated out-of-band, however, the threshold should
```

Comment:
Great clarification! :+1: 

I think we should add the point that, in the unlikely event where a threshold of root role keys have been compromised, then you have much bigger problems to worry about than key revocation. Here, it's safest to assume that attackers have installed malware and taken over affected machines. Now, key revocation and replacement should be assumed to be impossible, at least not without destroying the remote device and rebuilding everything from scratch.

What do you guys think?